subprojects/core-api/src/main/java/org/gradle/api/artifacts/ProjectDependency.java

```diff
@@ -28,6 +28,16 @@
 @HasInternalProtocol
 @SuppressWarnings("deprecation") // Because of SelfResolvingDependency
 public interface ProjectDependency extends ModuleDependency, SelfResolvingDependency {
+
+    /**
+     * Get the path to the project that this dependency refers to relative to its owning build.
+     *
+     * @see Project#getPath()
+     *
+     * @since 8.11
+     */
+    String getPath();
```

Comment:
🤔  Is there a way to obtain the build path as well? Would it be important to get the build path or is this not something you'd normally do?